docker-compose.yml

```
@@ -13,5 +13,10 @@ services:
       - NODE_ENV=development
       - CHOKIDAR_USEPOLLING=true
       - WATCHPACK_POLLING=true
+      - WEBSITE_DOMAIN=localhost
+      - DEV_PORT=3000
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://${WEBSITE_DOMAIN}:${DEV_PORT}"]
+      test: ["CMD", "curl", "-f", "http://localhost:3000"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)